app/pb_kits/playbook/pb_form/docs/_description.md

```diff
@@ -0,0 +1 @@
+Form displays containers with labels and fields for a user to fill out / input information. Typically used on cards or white backgrounds.
```

Comment:
@thestephenmarshall This kit is a bit technical. Can you help Megan provide a bit more detail about what this kit is? I think a bit more explanation about how this kit is used for forms in rails perhaps